src/views/studio/MyUploadsView/AssetsGroupUploadBar/AssetsGroupUploadBar.style.tsx

```diff
@@ -61,7 +62,12 @@ export const ProgressBar = styled.div<ProgressbarProps>`
   transform-origin: 0 0;
   transform: scaleX(${({ progress }) => progress && `${progress / 100}`});
   transition: transform 1s linear;
-  animation: ${pulse} 2.5s infinite ease-in-out;
+  animation: ${({ hasUploadingAsset }) =>
```

Comment:
I don't like this style of splitting the style declaration, let's keep it more concise. For example like this:
```ts
const pulseAnimationCss = (props: ProgressbarProps) =>
  props.hasUploadingAsset
    ? css`
        animation: ${pulse} 2.5s infinite ease-in-out;
      `
    : null

export const ProgressBar = styled.div<ProgressbarProps>`
  // ...
  ${pulseAnimationCss};
`